web/README.md

```diff
@@ -0,0 +1,242 @@
+# Rafiki Dashboard
+
+## How to run the app
```

Comment:
(Minor) Can include instructions on how to install YARN, how to install dependencies e.g. `yarn install`